yourgov/components/Staff/AccessRequestsTable/AccessRequestsTable.tsx

```diff
@@ -119,16 +118,23 @@ export const AccessRequestsTable = () => {
 
 	return (
 		<>
-			{successMessageContent && successMessageType && (
+			<Box
+				css={{
+					display:
+						successMessageContent && successMessageType ? 'block' : 'none',
```

Comment:
Is `inline-block` safer here as a reference for others who might restrict the width of the alert?

Reply:
The alerts should be full-width shouldn't they? They were before this change too?

Reply:
`SectionAlert`s spread their props to `Flex` and I thought I'd seen a consumer attempting smaller alerts but I may just have been confused. Probably not a real issue as I think about it more, especially as we evolve this into a pattern/component.